### ISSUE TITLE
Handling multiple users

### ISSUE THREAD

**Issue by libussa** [jellyfin/jellyfin-plugin-trakt#221]
Hi! 
How do you recommend handling multiple users to use the plugin? If I'm not mistaken, an admin needs to click on authorize, and give the short code to the user through a secondary channel.
Would there be a way to expose that config directly to the user, so that they can configure their trakt link themselves?

**Comment by Shadowghost** [comment]
Currently not possible due to how plugin config pages work.

**Comment by GlassedSilver** [comment]
Wow, I just noticed that plays scrobble for every user. It'd be INCREDIBLY helpful to put a big and bold disclaimer into the plugin's setup page, so everyone can make their own informed decision whether this affects them or not.

Is there an issue at the Jellyfin repo that tracks this limitation?

**Comment by alvitali** [comment]
> Yeah need multiple users please...

mhm. would indeed be a great feature to have though. while i'm not sure the current architecture of jellyfin allows for this, [having every user manage trakt themselves](https://forum.jellyfin.org/t-solved-setting-up-trakt-for-multiple-users) would be amazing.

i'm subscribing here but uninstalling the plugin for now, hopefully someone more capable can take a shot at it.

**Comment by skinnyzaz** [comment]
yes this is needed so badly, maybe an option under profile to authorize their trakt account.
